### PR TITLE
Fix bug in suggestion box

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
Read #171

### CODE REVIEW

File: run_dir/static/js/suggestion_box.js [hunk 1]
```diff
@@ -66,21 +69,18 @@ $.validator.setDefaults({
 //Generate alert when POSTing and refresh suggestions table
 $("#suggestionForm").validate({
   submitHandler: function(form) {
-    var spinnerText = "<div class='test_padding'> \
-                        <span class='icon-refresh glyphicon-refresh-animate'></span>  Processing your suggestion... \
-                      </div>"
-    $('#modalBody').html(spinnerText);
+    var spinnerText = "<div style=\"text-align:center; margin:20px 0;\"><span class=\"glyphicon glyphicon-refresh glyphicon-spin\"></span>  Processing your suggestion..</div>"
```
Comment (ewels) [inline run_dir/static/js/suggestion_box.js]: Minor thing, but backslashed quote marks bug me. Why not use single quotes for the string instead?

File: run_dir/static/js/suggestion_box.js [hunk 2]
```diff
@@ -66,21 +69,18 @@ $.validator.setDefaults({
...
+    $('#processingModalBody').html(spinnerText);
     $('#processingModal').modal('toggle');
     $.post('/suggestion_box', $('#suggestionForm').serialize())
     .done(function() {
       $("#modalBody").html("Suggestion processed correctly! <i class='icon-ok'></i>");
+      fill_suggestions_table();
+      document.getElementById('suggestionForm').reset();
```
Comment (ewels) [inline run_dir/static/js/suggestion_box.js]: Again, minor, but if you're using jQuery you may as well use jQuery:

```javascript
$("#suggestionForm").trigger('reset');
```

**Comment by ewels** [comment]
Nice! Sorry about not updating the modal template when I was updating everything else. I guess this one was my bad really..

**Comment by guillermo-carrasco** [comment]
No problem :-) 

I changed that `document.getElementById('suggestionForm').reset();` call

**Comment by guillermo-carrasco** [comment]
@ewels can I merge and deploy now? :-)

**Comment by ewels** [comment]
Great!

**Pull request merged**